src/data/lovelace/config/view.ts

```diff
@@ -18,6 +18,7 @@ export interface LovelaceBaseViewConfig {
   visible?: boolean | ShowViewConfig[];
   subview?: boolean;
   back_path?: string;
+  max_columns?: number; // Only used for section view
```

Comment:
Can we move this out of `LovelaceBaseViewConfig` then and make it a type that extends `LovelaceBaseViewConfig`?

Reply:
For now it's needed is the base view type because it's used in the generic editor. 
It's a quick fix for this highly requested feature. 
We will have dedicated editor for each view type in a future PR with dedicated type.